documentation/source/user_section/installation.rst

```diff
@@ -96,8 +96,125 @@ In the context of SCT, it can be used:
   virtual machine
 - <your reason here>
 
-See https://github.com/neuropoly/sct_docker for more information. We also provide a
-`tutorial to install SCT via Docker <https://github.com/neuropoly/spinalcordtoolbox/wiki/testing#run-docker-image>`_.
+Option 1: Without GUI
+=====================
+
+First, `install Docker <https://docs.docker.com/install/>`_. Then, follow the examples below to create an OS-specific SCT installation.
+
+
+Ubuntu-based installation
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: bash
+
+   # Start from the Terminal
+   docker pull ubuntu:16.04
+   # Launch interactive mode (command-line inside container)
+   docker run -it ubuntu
+   # Now, inside Docker container, install dependencies
+   apt-get update
+   yes | apt install git curl bzip2 libglib2.0-0 gcc
+   # Note for above: libglib2.0-0 is required by PyQt
+   # Install SCT
+   git clone https://github.com/neuropoly/spinalcordtoolbox.git sct
+   cd sct
+   yes | ./install_sct
+   export PATH="/sct/bin:${PATH}"
+   # Test SCT
+   sct_testing
+   # save the state of the container. Open a new Terminal and run:
+   docker ps -a  # list all containers
+   docker commit <CONTAINER_ID> <YOUR_NAME>/ubuntu:ubuntu16.04
+
+CentOS7-based installation
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: bash
+
+   # Start from the Terminal
+   docker pull centos:centos7
+   # Launch interactive mode (command-line inside container)
+   docker run -it centos:centos7
+   # Now, inside Docker container, install dependencies
+   yum install -y which gcc git curl
+   # Install SCT
+   git clone https://github.com/neuropoly/spinalcordtoolbox.git sct
+   cd sct
+   yes | ./install_sct
+   export PATH="/sct/bin:${PATH}"
+   # Test SCT
+   sct_testing
+   # save the state of the container. Open a new Terminal and run:
+   docker ps -a  # list all containers
+   docker commit <CONTAINER_ID> <YOUR_NAME>/centos:centos7
+
+
+Option 2: With GUI
```

Comment:
Naming it "option 2" suggests that option 1 doesn't need to be run, and I don't think that's true. Option 1 describes SCT installation (which is always needed, even if the GUI is required).

Reply:
My apologies, I had skimmed the instructions and misunderstood. Thanks for catching this.